lib/task/migrate/migrations/arMigration0153.class.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Access to Memory (AtoM) software.
+ *
+ * Access to Memory (AtoM) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Access to Memory (AtoM) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Access to Memory (AtoM).  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/*
+ * Add taxonomy and terms for job statuses
+ *
+ * @package    AccesstoMemory
+ * @subpackage migration
+ */
+class arMigration0153
+{
+  const
+    VERSION = 153, // The new database version
+    MIN_MILESTONE = 2; // The minimum milestone required
+
+  public function up($configuration)
+  {
+    QubitMigrate::bumpTaxonomy(QubitTaxonomy::JOB_STATUS_ID, $configuration);
+    $taxonomy = new QubitTaxonomy;
+    $taxonomy->id = QubitTaxonomy::JOB_STATUS_ID;
+    $taxonomy->parentId = QubitTaxonomy::ROOT_ID;
+    $taxonomy->sourceCulture = 'en';
+    $taxonomy->setName('Job statuses', array('culture' => 'en'));
```

Comment:
I would change name to be 'Job status' to match data/fixtures/taxonomyTerms.yml:

QubitTaxonomy_job_status:
    source_culture: en
    id: 79
    parent_id: QubitTaxonomy_root
    name:
      en: 'Job status'